tests/server-integration/Main.hs

```diff
@@ -24,8 +24,8 @@ main = do
           do
             setEnv "COFFER_CONFIG" "tests/server-integration/config.toml"
             server <- async runServer
-            (_, _, _, vault1) <- runVault 8209 "root"
-            (_, _, _, vault2) <- runVault 8211 "second"
+            (_, _, _, vault1) <- runVault 8212 "root"
```

Comment:
IIRC, vault may take up to two ports. So when you start it on port 1000, it may take up ports 1000 and 1001.

Since the bats tests use 8209 and 8211, using port 8212 here may not always work...

Let's use ports 8213 and 8215 here instead.

Reply:
done